### DISCUSSION TITLE
Skoda Enyaq IV

### DISCUSSION THREAD

**Discussion by mnicex** [tillsteinbach/VWsFriend#599] [Ideas]
This looks interesting, has anyone tried with Skoda Enyaq ?

**Comment by tillsteinbach** [comment]
Enya is not implemented (yet), but I ordered an Enyaq so there is a chance that I will add support this year

**Reply by AndaleR** [reply]
Are there any news?

I‘ve seen, you are looking for some data for the MySkoda-stuff?

**Reply by tillsteinbach** [reply]
Did not get my car yet ;)

**Reply by AndaleR** [reply]
I could share my MySkoda-Account or maybe create another user in the car for sharing?

**Reply by gitjpk** [reply]
Hello ! Thanks for the work. Did you have the chance to get the car and start the work on ?

**Reply by tillsteinbach** [reply]
I will get my Enyaq tomorrow. I hope I find some time over Christmas.

**Reply by gitjpk** [reply]
Great news ! Which one will you get ? 
I just received the 85x this week but snow in France reduce the opportunity to play with !

**Reply by tillsteinbach** [reply]
A white RS
